### PR TITLE
Mocha jest

### PR DESCRIPTION
**Pull request by meta-dipanshu-sharma**
# COMPLETES #< [SPARK-382919](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-382919) >

## This pull request addresses

Wrote jest.js for running test cases on jest environment except `plugin-meetings` and `webex` packages which still runs on mocha.
Reverted jest changes in plugin-meetings test files.

### Change Type

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested

< ENUMERATE TESTS PERFORMED, WHETHER MANUAL OR AUTOMATED >

### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [x] I discussed changes with code owners prior to submitting this pull request

- [x] I have not skipped any automated checks
- [ ] All existing and new tests passed
- [ ] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/@webex/internal-plugin-conversation/test/unit/spec/conversation.js [hunk 1]
```diff
@@ -135,7 +134,7 @@ describe('plugin-conversation', () => {
       it('should convert a "us" cluster to WEBEX_CONVERSATION_DEFAULT_CLUSTER cluster', async () => {
         await webex.internal.conversation.getUrlFromClusterId({cluster: 'us'});
 
-        sinon.assert.calledWith(webex.internal.services.getServiceFromClusterId, {clusterId: 'urn:TEAM:us-east-2_a:identityLookup'});
+        sinon.assert.calledWith(webex.internal.services.getServiceFromClusterId, {clusterId: 'urn:TEAM:us-east-1_int13:identityLookup'});
```
Comment (arun3528) [inline packages/@webex/internal-plugin-conversation/test/unit/spec/conversation.js:137]: why this change ?

File: tooling/commands/test.js [hunk 2]
```diff
@@ -156,7 +156,9 @@ module.exports = {
 
         /** Each package is run through testPackage util */
         for (const packageName of argv.packages) {
-          await testPackage(argv, packageName);
+          const onMocha = packageName === '@webex/plugin-meetings' || packageName === 'webex';
```
Comment (arun3528) [inline tooling/commands/test.js:159]: Hope this does not grows , But if it does lets move the names to a array constant

**Comment by arun3528** [comment]
i also dont see PR checks running , we can merge this and check the jest branch

File: packages/@webex/internal-plugin-conversation/test/unit/spec/conversation.js (hunk 1 above)
Comment (meta-dipanshu-sharma) [reply packages/@webex/internal-plugin-conversation/test/unit/spec/conversation.js:137]: This test was failing so fixed it with this change.

File: tooling/commands/test.js (hunk 2 above)
Comment (meta-dipanshu-sharma) [reply tooling/commands/test.js:159]: Yeah, for now jest is running good on other packages than these two.
